pyproject.toml

```diff
@@ -1,88 +1,44 @@
-    "plugins/*/*/*/*.txt",
-    "plugins/*/*/*/*/*.txt",
-]
-
-[tool.setuptools.dynamic]
-entry-points = {file = ["entry-points.ini"]}
-
-[tool.setuptools_scm]
-"version_scheme" = "post-release"  # Use current version .postN vs incrementing
-"local_scheme" = "no-local-version"  # Don't include hash, or date, just postN
-# @ Update with your package name, ie:
-# @ "write_to" = "@package@/_version.py"  # Write an actual file
-"write_to" = "geoips_proxy_vis/_version.py"  # Write an actual file
-
-[project]
-# @ Update with your package name, ie:
-# @ name = "@package@"
-name = "geoips_proxy_vis"
+[tool.poetry]
+name = "geoips_proxy_vis"                                          # geoips package available at the root of the project
+version = "0.0.0"
+description = "geoips_proxy_vis GeoIPS Plugin Package"
 authors = []
-description = "@package@ GeoIPS Plugin Package"
 readme = "README.md"
-requires-python = ">=3.9"
 keywords = []
-license = {file = "LICENSE"}
-classifiers = [
+license = "LICENSE"   # required
+classifiers = [ # // list of PyPI trove classifiers to describe the project
     "Programming Language :: Python :: 3",
 ]
-dynamic = ["version"]
+repository = "https://github.com/robertdemariacira/geoips_proxy_vis" #optional
+#documentation = "set the url"
+include = ["**/*.txt", "**/*.py", "**/*.yaml"]
+
+# uses current version .postN and not incrementing
+[tool.poetry-dynamic-versioning]
+enable = true
+vcs = "git"
+metadata = false # don't include local-version hash; date doesn't appear by deafult
+style = "pep440" # pep440 is default; can rmeove later on; used for : keeping 0.0.8 as the default install until 0.0.9 is out
+#style = "semver"
+#tag-branch = "feature-poetry"
+strict = false
+pattern = "(?x)^((?P<epoch>\\d+)!)?(?P<base>\\d+(\\.\\d+)*)([-._]?((?P<stage>[a-zA-Z]+)[-._]?(?P<revision>\\d+)?))?(\\+(?P<tagged_metadata>.+))?$"
+
+[tool.poetry-dynamic-versioning.substitution]
+files = ["*/_version.py"]
+
+# This is where you would specify a pre-build script from pyproject.toml/poetry.
+# [tool.poetry.build]
+# generate-setup-file = false  # I have no idea what this does
+# script = "build.py"  # This gets called as a python script prior to running build steps
 
-[project.entry-points."geoips.plugin_packages"]
-# @ Update with your package name, ie:
-# @ "@package@" = "@package@"
-"geoips_proxy_vis" = "geoips_proxy_vis"
-
-[project.entry-points."geoips.algorithms"]
-"pmw_89test" = "geoips_proxy_vis.plugins.modules.algorithms.pmw_89test"
-"geo_proxy_vis" = "geoips_proxy_vis.plugins.modules.algorithms.geo_proxy_vis"
+[build-system]
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
+build-backend = "poetry_dynamic_versioning.backend"                          # object used to perform the build process
 
+[tool.poetry.dependencies] # must download to run
+python = ">=3.10"       # mandatory to declare the required python version
+proxy_vis = { git = "https://github.com/galina-cira/ProxyVis" }
```

Comment:
Added this to pyproject.toml so it automatically gets installed when you pip install geoips_proxy_vis